src/panels/lovelace/common/create-hui-element.js

```diff
@@ -0,0 +1,46 @@
+import '../elements/hui-icon-element.js';
+import '../elements/hui-image-element.js';
+import '../elements/hui-service-button-element.js';
+import '../elements/hui-state-badge-element.js';
+import '../elements/hui-state-icon-element.js';
+import '../elements/hui-state-label-element.js';
+
+import createErrorCardConfig from './create-error-card-config.js';
+
+const ELEMENT_TYPES = new Set([
+  'icon',
+  'image',
+  'service-button',
+  'state-badge',
+  'state-icon',
+  'state-label',
+]);
+
+function _createElement(tag, config) {
+  const element = document.createElement(tag);
+  try {
+    element.setConfig(config);
+  } catch (err) {
+    // eslint-disable-next-line
+    console.error(tag, err);
+    // eslint-disable-next-line
```

Comment:
What is disabled about next line ?

Reply:
`no-use-before-define` on `_createErrorElement()` since it's defined later in the file